### PR TITLE
chore(docs) - Help clarify the difference between Custom Profiles and…

### PR DESCRIPTION
**Pull request by clanesf**
… Service-Settings and explain which kinds of configurations go where

### CODE REVIEW

File: reference/halyard/custom.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ configuration into Halyard. For these users & use-cases, Halyard supports
 custom [Profiles](/reference/halyard/#profiles), and custom [Service
 Settings](/reference/halyard/#service-settings).
 
+## Custom Profiles versus Service settings
+
+The two most common ways of customizing your Spinnaker installation is through the use of either a Custom profile, for example `clouddriver-local.yml` or a Service Setting, for example `clouddriver.yml`. Each of these are created in separate subfolders inside the main Halyard deployment profile folder. In a `default` Halyard profile, this means profiles go in `.hal/default/profiles/` while service-settings go in `.hal/default/service-settings`.
```
Comment (dorbin) [inline reference/halyard/custom.md]:
```suggestion
The two most common ways of customizing your Spinnaker installation is through the use of either a custom profile, for example `clouddriver-local.yml`, or a service setting, for example `clouddriver.yml`. Each of these is created in separate subfolders inside the main Halyard deployment profile folder. In a `default` Halyard profile, this means profiles go in `.hal/default/profiles/` while service settings go in `.hal/default/service-settings`.
```

File: reference/halyard/custom.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ configuration into Halyard. For these users & use-cases, Halyard supports
...
+
+ Custom profiles are configuration files that override the default configuration files for each Spinnaker Service. Service-Settings are used by Halyard to determine how to build the Spinnaker deployment and contain information specific to how each Spinnaker Service will be deployed.
```
Comment (dorbin) [inline reference/halyard/custom.md:23]:
```suggestion
 Custom profiles are configuration files that override the default configuration files for each Spinnaker service. Service settings are used by Halyard to determine how to build the Spinnaker deployment, and they contain information specific to how each Spinnaker service will be deployed.
```

File: reference/halyard/custom.md [hunk 3]
```diff
@@ -16,6 +16,16 @@ configuration into Halyard. For these users & use-cases, Halyard supports
...
+
+For example, if you want to substitute which image file will be used for a Spinnaker Service, or what environment variables the service will be run with, then you'll need to modify the Service-Settings for that Service.
```
Comment (dorbin) [inline reference/halyard/custom.md]:
```suggestion
For example, if you want to substitute which image file will be used for a Spinnaker service, or what environment variables the service will be run with, then you'll need to modify the service settings for that service.
```

File: reference/halyard/custom.md [hunk 4]
```diff
@@ -16,6 +16,16 @@ configuration into Halyard. For these users & use-cases, Halyard supports
...
+
+If you want to override the level of debugging that is specified by default inside the clouddriver.yml configuration, then you'll need to create a custom clouddriver-local.yml file and put it in the Profiles directory.
```
Comment (dorbin) [inline reference/halyard/custom.md]:
```suggestion
To override the debugging level specified in `clouddriver.yml`, create a custom `clouddriver-local.yml` file and put it in the `/profiles/` directory.
```

**Review by dorbin — COMMENTED** [review]
Some suggested edits, @clanesf . Otherwise looks good.

**Comment by clanesf** [comment]
Agreed @dorbin, thanks for the cleanup.

**Review by dorbin — APPROVED** [review]
Thanks, @clanesf !

**Pull request merged**